Bixal/move.mil
# Feature 156665114 updates to tutorial theming

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- adds specificity to ul elements with the class .move-mil--tutorial__steps so as to not override other ul elements on the same node

## Testing

To verify the changes proposed in this pull request…

1. pull changes locally
2. cd /move-mil/web/themes/custom/move_mil
3. npm run build
4. make cr
5. navigate to /tutorials
6. edit first slide to include bullets in the protip, save 

you should see that protips now renders the ul stiles for lists

## Screenshots
![screen shot 2018-04-18 at 3 34 12 pm](https://user-images.githubusercontent.com/37077057/38954035-91ba5ebe-431e-11e8-99af-970b49d96e41.png)
